Formula/libdill.rb

```diff
@@ -0,0 +1,54 @@
+class Libdill < Formula
+  desc "Structured concurrency in C"
+  homepage "http://libdill.org"
+  url "https://github.com/sustrik/libdill/archive/1.6.tar.gz"
+  sha256 "2539d3440971c876b117bc87295b0caabc8c8ec5f544856ca9e74ed12705ea51"
+
+  option "without-sockets", "Compile without socket support"
+  option "without-threads", "Compile without threading support"
+  option "with-valgrind", "Compile with valgrind compatibility"
```

Comment:
are you yourself going to use this? if not, I'd rather leave it out

Reply:
Any particular reason why you'd prefer to exclude it?

Reply:
because valgrind breaks during every macOS upgrade :see_no_evil:

Reply:
Fair enough. [I did manage to get valgrind working on High Sierra, but nobody's said anything there yet.](https://github.com/Homebrew/homebrew-core/issues/18998#issuecomment-354189409)